demisto_sdk/commands/download/tests/downloader_test.py

```diff
@@ -1532,3 +1532,32 @@ def test_invalid_regex_error(mocker):
         logger_error.call_args_list,
         "Error: Invalid regex pattern provided: '*invalid-regex*'.",
     )
+
+
+def test_download_with_subplaybook(mocker):
+    """
+    Given: A downloader object
+    When: Downloading custom a playbook with task of subplaybbok.
```

Comment:
I think this line should be:
```suggestion
    When: Downloading a custom playbook with task of subplaybook.
```